deploy-asm.sh

```diff
@@ -33,40 +33,30 @@ echo "No CLUSTERNAME variable set"
 exit
 fi
 
-gcloud services enable cloudtrace.googleapis.com
-
-gcloud container clusters get-credentials $CLUSTERNAME --project=$PROJECT --zone=$LOCATION
-kubectl config set-context $CLUSTERNAME
-
-echo "Installing ASM..."
-curl -s https://storage.googleapis.com/csm-artifacts/asm/asmcli_1.13 > asmcli
-chmod +x asmcli
+echo "Register GKE cluster to the fleet..."
+gcloud container fleet memberships register $CLUSTERNAME \
+    --gke-cluster $LOCATION/$CLUSTERNAME \
+    --enable-workload-identity
+    
+echo "Enable Managed ASM on the project..."
+gcloud container fleet mesh enable
```

Comment:
Nop, this part is only to say to the project that potentially some clusters in it will potentially have ASM installed on them. Does that make sense? Any concern with that?